src/js/components/pages/scene-media-browser.jsx

```diff
@@ -70,16 +72,16 @@ var SceneMediaBrowser = React.createClass({
             <div className='RemoveWidgetPadding'>
                 <div className={showOverlay}></div>
                 <Measure onMeasure={this.onMeasure}>
-                    <SplitPane id="splitPane" split="horizontal" minSize={0} defaultSize={this.getLastSplit()} onChange={size => this.splitChanged(size)}>
-                        <div style={{height: "100%", width: "100%"}}>
-                            <MediaPreviewComponent 
-                                style={{height: "100%", width: "100%"}} 
+                    <SplitPane id="splitPane" split="horizontal" minSize={0} maxSize={-15} defaultSize={this.getLastSplit()} onChange={size => this.splitChanged(size)}>
+                        <div style={{width: "100%"}}>
```

Comment:
Need to determine if this removal of height: 100% has any implications.

Tests so far don't show any issue.